### DISCUSSION TITLE
fa/manager/configuration-and-advanced-settings/How-to-set-up-and-use-Telegram-bot-on-Hiddify-panel/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#3960] [Content Help کمک به اصلاح محتوا]
# fa/manager/configuration-and-advanced-settings/How-to-set-up-and-use-Telegram-bot-on-Hiddify-panel/

The proxy manager platform.

https://hiddify.com/fa/manager/configuration-and-advanced-settings/How-to-set-up-and-use-Telegram-bot-on-Hiddify-panel/

**Comment by moli1369** [comment]
سلام لطفا کاملش کنید به شدت احساس میشه.

**Comment by mokhtarabdi** [comment]
من دوتا پنل دارم یکیشون ربات ران میشه ولی یکیشون ربات فقط دستور بکاپ رو اجرا میکنه

**Comment by JuniorLOL** [comment]
Hello there Thanks for the awesome manager its really easy to use. I"m new in coding but i wanna create my `custom` panel if it"s possible, to be precise i want to add more features to Tgbot but for this moment bot has only one function add user and i can"t find anywhere in project where its describes the logic of the bot's operation. Can you pinpoint to me. Appreciate

**Comment by cnakJS** [comment]
لطفا اموزش اتصال این بات به درگاه پرداخت رو هم بزارید
